packages/richtext-lexical/src/features/blocks/client/component/index.scss

```diff
@@ -1,34 +1,30 @@
 @import '~@payloadcms/ui/scss';
 
 @layer payload-default {
-  .ContentEditable__root > div:has(.lexical-block) {
+  .LexicalEditorTheme__block {
+    @extend %body;
+    margin-block: base(0.4);
```

Comment:
If a user changes the class, would all the styles in our admin panel still be in effect? ​​How?